youtube_dl/jsinterp/__init__.py

```diff
@@ -1,3 +1,5 @@
 from .jsinterp import JSInterpreter
+from .jsgrammar import _NAME_RE
 
-__all__ = ['JSInterpreter']
\ No newline at end of file
+# ALERT stop usage of _NAME_RE!
```

Comment:
openload is broken again (#10408) and most likely it requires substantial rewriting. You can just remove _NAME_RE from openload.py.